airflow_example/dags/tableau_datasource_update/tableau_datasource_update.py

```diff
@@ -11,14 +11,24 @@
 import re
 import ast
 from copy import deepcopy
-from tdscc import TDS, TableauServer, extract_tds, update_tdsx, TDSCCError
+from tableau_utilities import TDS, TableauServer, extract_tds, update_tdsx, TableauUtilitiesError
 
 from airflow import DAG, models
 from airflow.operators.python_operator import PythonOperator
 from airflow.hooks.base_hook import BaseHook
+from airflow.models import Variable
 
 
-CFG_PATH = 'dags/tableau_datasource_update/configs'
+SNOWFLAKE_WAREHOUSE = Variable.get('snowflake_warehouse')
+SNOWFLAKE_DATABASE = Variable.get('snowflake_database')
+SNOWFLAKE_SCHEMA = Variable.get('snowflake_schema')
+SNOWFLAKE_ROLE = Variable.get('snowflake_role')
+SNOWFLAKE_USER = Variable.get('snowflake_user')
+SNOWFLAKE_PASSWORD = Variable.get('snowflake_password')
+SNOWFLAKE_URL = Variable.get('snowflake_url')
+
+
+CFG_PATH = 'dags/tableau/configs'
```

Comment:
Can we revert these?

Reply:
@JustinGrilli  I wasn't able to get the DAG to work with connections so I made these independent variables for ease of troubleshooting. I can revert these after I test but that won't be for a bit

Reply:
Cool, no rush :)
Thanks!

Reply:
@jaybythebay We recently upgrade to Airflow 2 as well, so we've got some insight about getting these Snowflake connections set, if that helps:
You can set these extra attributes, by adding them to a connection in the `extra` section, and by prefixing each attribute with `extra__snowflake__`. i.g. `{"extra__snowflake__role": "tableau_role"}`.

Here is a sample of how it would look in `airflow_settings.yaml`:
```yaml
    - conn_id: snowflake_tableau_datasource
      conn_type: snowflake
      conn_host: snowflake_account_name.here
      conn_schema: tableau_schema
      conn_login: tableau_user
      conn_password: password
      conn_extra: '{
        "extra__snowflake__account": "snowflake_account_name.here",
        "extra__snowflake__database": "tableau_database",
        "extra__snowflake__schema": "tableau_schema",
        "extra__snowflake__role": "tableau_role",
        "extra__snowflake__warehouse": "tableau_warehouse"
      }'
```